tests/mlmodel_openai/test_embeddings_v1.py

```diff
@@ -81,6 +91,26 @@ def test_openai_embedding_sync(set_trace_info, sync_openai_client):
     sync_openai_client.embeddings.create(input="This is an embedding test.", model="text-embedding-ada-002")
 
 
+@reset_core_stats_engine()
+@override_application_settings({"ai_monitoring.record_content.enabled": False})
```

Comment:
Instead of calling override application settings everywhere like this. Do the same thing we did with disabled_ai_monitoring_settings in conftest for all these frameworks. It's just simpler to write it once and use it in all the test files than re-implementing it every time.